.cirrus.yml

```diff
@@ -0,0 +1,22 @@
+aliases:
+  - &bundle_cache
```

Comment:
I think the syntax should be like this:

```suggestion
  bundle_cache: &bundle_cache |
```

Reply:
Why (for what) `|`?

Reply:
Still no…

Reply:
`|` for multiline strings, as I know.

Aliases are correct, and parsed (in Ruby) correctly.

Reply:
You can check YAML file here, for example: https://yaml-online-parser.appspot.com/